src/components/ListView.js

```diff
@@ -1,10 +1,22 @@
 import React from 'react';
+import { db } from '../lib/firebase';
+import { useCollectionData } from 'react-firebase-hooks/firestore';
 
-const ListView = () => {
+const ListView = ({ token }) => {
+  let tokenValue = token ? token : 'LA';
```

Comment:
Being curious... What does  'LA' stand for here?

Reply:
I have renamed it. Sorry for the confusion @katia-utochkina